embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/internal/network/http/EmbraceHttpUrlConnectionOverride.java

```diff
@@ -28,7 +28,7 @@ public String getHeaderByName(@NonNull String name) {
     public String getOverriddenURL(@NonNull String pathOverride) {
         try {
             return new URL(connection.getURL().getProtocol(), connection.getURL().getHost(),
-                connection.getURL().getPort(), pathOverride).toString();
+                connection.getURL().getPort(), pathOverride + "?" + connection.getURL().getQuery()).toString();
```

Comment:
What's the purpose of this change? Is it because we were discarding the query params?

Reply:
Yeah. This is the bug I mention in the second bullet point in the PR description.  When you `toString()` a `URL`, it includes the query path as part of the returned string. When we return teh URL with an overridden path, we should do the same. (This is what the OkHttp implementation does)